src/main/java/org/balanpy/PerfilUsuario.java

```diff
@@ -24,6 +30,24 @@ public class PerfilUsuario {
 	@FXML
 	private Label direccion;
 
+	public void inicialize() {
+
+		Usuario usuario = UsuarioImpl.getInstance();
+
+		nombre.setText(usuario.getNombre());
+		edad.setText(String.valueOf(usuario.getEdad()));
```

Comment:
Esto está bien, pero siendo que edad es un `Integer`, creo que esta forma se lee un poco mejor:
```suggestion
		edad.setText(usuario.getEdad().toString());
```

Reply:
Perfec, resuelto